.github/workflows/lint.yml

```diff
@@ -33,7 +33,9 @@ jobs:
         env:
             cache-name: cache-pnpm-modules
         with:
-            path: ~/.pnpm-store
+            path:  |
+                    ~/.pnpm-store
+                    **/node_modules/.cache/nx/
```

Comment:
Hi there!

It's likely that with Nx Cloud enabled you won't actually want to do this -- there isn't any sort of age-based cache pruning in Nx so this folder will just swell in size and slow down your pipelines on cache retrieval.

Reply:
Thanks. It was a temporary approach.  
Reverted in https://github.com/SAP/open-ux-tools/pull/799/commits/55cb1edd79e69741e810fddce4fb50dd8634ca34